### PR TITLE
IE working and button colors fixed

### PR DESCRIPTION
**Pull request by Georgehe4**
- IE works (load time 1.2~1.5s from what I can observe - same as dev.dartpad)
- Removed unnecessary file links.
- Imports.html is now vulcanized. Is it possible to cache this so we don't request it every time? It is a 500kb file.
- Buttons have been fixed (color switch is more subtle - can also have no color switch if preferred). Export button is BLUE until further suggestions. -- Switched to grey
- IMAGES do not load in properly still. -- Fixed

Test site at: http://dev.dartproject-1000.appspot.com/embed-dart.html

@devoncarew

### CODE REVIEW

**Comment by Georgehe4** [comment]
#632

File: tool/grind.dart [hunk 1]
```diff
@@ -89,6 +90,28 @@ vulcanize(String filepath) {
...
+    '--exclude',
+    'embed_components.html',
+    '--exclude',
+    'animated-dropdown.html',
+    filepath
+  ], workingDirectory: 'build/web');
+  if (result.exitCode != 0) {
+    fail('error running vulcanize: ${result.exitCode}\n${result.stderr}');
+  }
+  HtmlFile.asFile.writeAsStringSync(result.stdout);
```
Comment (devoncarew) [inline tool/grind.dart]: Can you change this var reference here and at the definition (`HtmlFile` ==> `htmlFile`)?

**Comment by devoncarew** [comment]
@Georgehe4, awesome work!

@lukechurch, I remember that our python front-end may have had issues w/ not reporting resources as cacheable - is this still the case? Or was that just for `index.html` / `mobile.html`?

I think we're drawing too much attention to the 2nd button:

screen shot 2015-08-13 at 10 11 55 am

Perhaps a gray color - something that stands out less. Run is still good w/ the accent color.

**Comment by devoncarew** [comment]
Also, let's tackle the image loading today; should go quickly.

**Comment by devoncarew** [comment]
lgtm

**Comment by Georgehe4** [comment]
Changes made. Image loading now works and export button now grey.
Merging.
TODO: Figure out caching.

**Pull request merged**